tests/network/test_node_storage.py

```diff
@@ -41,9 +43,9 @@ def start_lonely_learning_loop():
         newcomer.start_learning_loop()
         start = maya.now()
         # Loop until the_chosen_seednode is in storage.
-        while not the_chosen_seednode in newcomer.node_storage.all(federated_only=True):
+        while the_chosen_seednode not in newcomer.node_storage.all(federated_only=True):
             passed = maya.now() - start
-            if passed.seconds > 2:
+            if passed.seconds > 5:
```

Comment:
How can this ever take 5 seconds?  I thought 2 was already too long.

If it's actually a problem of time, then something else is wrong.

Reply:
Whoops! I was experiencing some Issues, I changed this for debugging.  Ultimately this is an unintended change.